.github/workflows/changelog_to_tag.yml

```
@@ -62,25 +62,24 @@ jobs:
               git branch -a
               exit 1
           fi
-          echo ::set-output name=tagname::"$_tagname"
-          echo ::set-output name=branch::"$_branch"
+          echo tagname="$_tagname" >> "$GITHUB_OUTPUT"
```

I prefer the following form. It seems a bit cleaner. But It could be merged as it is. :+1: 

```suggestion
          echo "tagname=${_tagname}" >> "$GITHUB_OUTPUT"
```